docs/websocket.rst

```diff
@@ -0,0 +1,32 @@
+.. _websocket:
+
+=========
+Websocket
+=========
+
+You can enable web sockets if you select ``use_async`` option when creating a project. That indicates whether the project should use web sockets with Uvicorn + Gunicorn.
+
+Usage
+-----
+
+JavaScript example: ::
+
+    > ws = new WebSocket('ws://localhost:8000/') // or 'wss://<mydomain.com>/' in prod
+    WebSocket {url: "ws://localhost:8000/", readyState: 0, bufferedAmount: 0, onopen: null, onerror: null, …}
+    > ws.onmessage = event => console.log(event.data)
+    event => console.log(event.data)
+    > ws.send("ping")
+    undefined
+    pong!
+
+If you are using nGinx instead of Traefik, you should add these lines to nGinx config: ::
```

Comment:
Why are our writing it `nGinx`? Doesn't look like a thing [according to Wikipedia](https://en.wikipedia.org/wiki/Nginx). I would stick to `Nginx`, unless you can quote a source supporting your spelling.